ArduSub/ArduSub.cpp

```diff
@@ -36,6 +36,7 @@ const AP_Scheduler::Task Sub::scheduler_tasks[] = {
     SCHED_TASK(three_hz_loop,          3,     75),
     SCHED_TASK(update_turn_counter,   10,     50),
     SCHED_TASK(compass_accumulate,   100,    100),
+    SCHED_TASK(update_beacon,         50,     50),
```

Comment:
@muramura -san, Thank you  for reviewing. I thought it is enough for Sub because Sub is not in air. The Hz is same to rover.